frontstage/controllers/party_controller.py

```diff
@@ -267,3 +267,18 @@ def change_respondent_status(respondent_id, status):
         raise ApiError(logger, response)
 
     logger.info('Successfully changed account status', respondent_id=respondent_id, status=status)
+
+
+def notify_account_locked(username):
+    logger.debug('Sending notification email for password reset')
+    url = f'{app.config["PARTY_URL"]}/party-api/v1/respondents/notify-respondent'
```

Comment:
This pr probably shouldn't be merged in until the party service changes have been made and merged.